frontend/main.js

```diff
@@ -175,6 +240,10 @@ async function startApp () {
     }
   })
 
+  sbp('okTurtles.events/on', EVENT_HANDLED, (contractID, message) => {
+    sbp('okTurtles.events/emit', SHELTER_EVENT_HANDLED, contractID, message)
+  })
+
```

Comment:
Weird code: why do this instead of just listening for `EVENT_HANDLED`?

If there's a good reason, it needs to be explained in a comment.

Reply:
Because this is a new event I added that Cypress listens for. The commit I made today also adds more code to this particular handler (as we discussed; it deals with keeping Chelonia and Vuex state in sync) that I'd hoped I wouldn't need for this PR.

Reply:
I don't think that answers my question as far as why Cypress can't listen directly for `EVENT_HANDLED`...

Reply:
Because Cypress needs to listen for when the Vuex state changed, which happens on SHELTER_EVENT_HANDLED.